manifests/init.pp

```diff
@@ -159,13 +159,15 @@
   if $fpm {
     Anchor['php::begin'] ->
       class { '::php::fpm':
+        ensure           => $ensure,
```

Comment:
@3flex I've opened https://github.com/voxpupuli/puppet-php/pull/248 with a possible way we could go about refactoring.  Let me know what you think.